plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -168,20 +212,20 @@ public function test_img_sizes_is_equal_to_picture_source_sizes_for_picture_elem
 
 		$img_markup = apply_filters( 'the_content', $image );
```

Comment:
This is also incorrectly returning JPEG. This method is also missing `$this->mock_frontend_body_hooks();`.